SwrveConversationSDK/Conversation/SwrveContentStarRatingView.m

```diff
@@ -42,7 +42,7 @@ - (id) initWithDefaults {
         self.swrveMaxRating = 5;
         self.swrveMidMargin = 5.0f;
         self.swrveLeftMargin = 0.0f;
-        self.swrveMinImageSize = CGSizeMake(15, 15);
+        self.swrveMinImageSize = CGSizeMake(40, 40);
```

Comment:
A minimum Size of 15 was too small and would never actually happen. 40 is an appropriate minimum size